src/ado2gh/Commands/GenerateScriptCommand.cs

```diff
@@ -600,6 +619,11 @@ private void AppendLine(StringBuilder sb, string content)
 
         private string WaitForMigrationScript(string repoMigrationKey) => $"./ado2gh wait-for-migration --migration-id $RepoMigrations[\"{repoMigrationKey}\"]";
 
+        private string DownloadMigrationLogScript(string githubOrg, string githubRepo)
+        {
+            return $"./ado2gh download-logs --github-org \"{githubOrg}\" --github-repo \"{githubRepo}\"";
```

Comment:
I think we should but a conditional in here like all the functions above it, where it checks if _generateScriptOptions.DownloadMigrationLogs is set (then we can remove all the other conditionals used elsewhere)

Reply:
Added in b1065614fe55041e29a8ae29edfdc420fc22daea!

I left a conditional for the parallel script generation, since it appends some spaces before the command:

https://github.com/github/gh-gei/blob/b1065614fe55041e29a8ae29edfdc420fc22daea/src/ado2gh/Commands/GenerateScriptCommand.cs#L476-L485